src/modules/user/entry-points/http/user.controller.ts

```diff
@@ -16,4 +19,38 @@ export class UserController {
       user,
     };
   }
+
```

Comment:
I know you asked me this before, but let's add a `@Get("me/wallets")` handler here and the response body can be like
```
{
  walletAddress: "<USER_WALLET_ADDRESS_HERE>"
}
```

This can also be the response of the POST and PATCH requests

Reply:
done